src/System.Private.Xml/src/System/Xml/XmlQualifiedName.cs

```diff
@@ -10,11 +10,13 @@ namespace System.Xml
     /// <devdoc>
     ///    <para>[To be supplied.]</para>
     /// </devdoc>
+    [Serializable]
     public class XmlQualifiedName
     {
         private string _name;
         private string _ns;
 
+        [NonSerialized]
```

Comment:
>NonSerialized [](start = 9, length = 13)

would your tool flag any places missing this (or that have it and shouldn't)
I didn't check.